ngraph/test/backend/aliased_output.in.cpp

```diff
@@ -38,32 +36,20 @@ NGRAPH_TEST(${BACKEND_NAME}, aliased_output)
     auto E = op::Constant::create(element::f32, shape, {1, 2, 3, 4});
     auto f = make_shared<Function>(NodeVector{C, C, D, D, C, E, E}, ParameterVector{A, B});
 
-    auto backend = runtime::Backend::create("${BACKEND_NAME}");
-
-    // Create some tensors for input/output
-    shared_ptr<runtime::Tensor> a = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> b = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out1 = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out2 = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out3 = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out4 = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out5 = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out6 = backend->create_tensor(element::f32, shape);
-    shared_ptr<runtime::Tensor> out7 = backend->create_tensor(element::f32, shape);
-
-    copy_data(a, vector<float>{0, 1, 2, 3});
-    copy_data(b, vector<float>{1, 2, 3, 4});
+    vector<float> a{0, 1, 2, 3};
+    vector<float> b{1, 2, 3, 4};
     vector<float> expectedC{1, 3, 5, 7};
     vector<float> expectedD{0, 2, 6, 12};
     vector<float> expectedE{1, 2, 3, 4};
 
-    auto handle = backend->compile(f);
-    handle->call_with_validate({out1, out2, out3, out4, out5, out6, out7}, {a, b});
-    EXPECT_TRUE(test::all_close_f(expectedC, read_vector<float>(out1), MIN_FLOAT_TOLERANCE_BITS));
-    EXPECT_TRUE(test::all_close_f(expectedC, read_vector<float>(out2), MIN_FLOAT_TOLERANCE_BITS));
-    EXPECT_TRUE(test::all_close_f(expectedD, read_vector<float>(out3), MIN_FLOAT_TOLERANCE_BITS));
-    EXPECT_TRUE(test::all_close_f(expectedD, read_vector<float>(out4), MIN_FLOAT_TOLERANCE_BITS));
-    EXPECT_TRUE(test::all_close_f(expectedC, read_vector<float>(out5), MIN_FLOAT_TOLERANCE_BITS));
-    EXPECT_TRUE(test::all_close_f(expectedE, read_vector<float>(out6), MIN_FLOAT_TOLERANCE_BITS));
-    EXPECT_TRUE(test::all_close_f(expectedE, read_vector<float>(out7), MIN_FLOAT_TOLERANCE_BITS));
+    auto test_case = test::TestCase<TestEngine>(f);
+    test_case.add_multiple_inputs<float>({a, b});
+    test_case.add_expected_output<float>(shape, expectedC);
+    test_case.add_expected_output<float>(shape, expectedC);
+    test_case.add_expected_output<float>(shape, expectedD);
+    test_case.add_expected_output<float>(shape, expectedD);
+    test_case.add_expected_output<float>(shape, expectedC);
+    test_case.add_expected_output<float>(shape, expectedE);
+    test_case.add_expected_output<float>(shape, expectedE);
+    test_case.run();
```

Comment:
run() will trigger with default value of DEFAULT_FLOAT_TOLERANCE_BITS. Here MIN_FLOAT_TOLERANCE_BITS is required.